EpiResponsivePicture/TagBuilders/ISourceTagBuilder.cs

```diff
@@ -0,0 +1,16 @@
+using Forte.EpiResponsivePicture.ResizedImage;
+using Forte.EpiResponsivePicture.ResizedImage.Property;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Forte.EpiResponsivePicture.TagBuilders
+{
+    public interface ISourceTagBuilder
+    {
+        ISourceTagBuilder WithImageUrl(string url);
+        ISourceTagBuilder WithSource(PictureSource source);
+        ISourceTagBuilder WithFocalPoint(FocalPoint point);
+        ISourceTagBuilder WithResizedImageFormat(ResizedImageFormat format);
+        TagBuilder Build();
+        ISourceTagBuilder Clear();
```

Comment:
you added Clear to `ISourceTagBuilder`. Shouldn't we add, for the sake of consistency, add Clear method to `IPictureTagBuilder` as well?